### PR TITLE
Issue/4323 e2e scenario 2.4

### LINKED ISSUE #4323: [E2E] Scenario 2.4 Service Catalog - Update
## Note
Before scenario make sure to execute the API call to clear the test-environment AND update call with the force flag set to true. 

## 2.4 Update Service catalog
- Select the environment named 'test'
- click on Update Service Catalog
- expect modal to pop
- click on cancel
- no toast should be displayed
- click again on Update Service Catalog
- Confirm modal
- Expect toast to be displayed  => Success alert:The update has been requested
- Go to Compile report page
- expect last compile to have message "Recompile model to export service definition"

### PR DESCRIPTION
**Pull request by matborowczyk**
# Description

* Short description here *

closes #4323 

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [ ] Attached issue to pull request
- [ ] Changelog entry
- [ ] Code is clear and sufficiently documented
- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Correct, in line with design
~~- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )~~

### CODE REVIEW

File: cypress/e2e/scenario-1-environment.cy.js [hunk 1]
```diff
@@ -112,13 +112,9 @@ describe("Environment", () => {
...
     cy.get('[aria-label="Name-input"]').type(testName(2));
     cy.get("button").contains("Submit").click();
-    cy.wait("@createEnv", { timeout: 10000 })
-      .its("response.statusCode")
-      .should("eq", 200);
-    cy.wait("@getCatalog");
     //go back to gome and check if env is visible
-
-    cy.get(".pf-c-breadcrumb__item").contains("Home").click();
+    cy.wait(500);
```
Comment (LauraStordeur) [inline cypress/e2e/scenario-1-environment.cy.js:116]: `cy.get('[aria-label="Environment card"]', { timeout: 20000))....` The wait can end up being flakey I fear on Jenkins

File: cypress/e2e/scenario-1-environment.cy.js [hunk 2]
```diff
@@ -166,56 +155,49 @@ describe("Environment", () => {
     cy.get('[aria-label="Name-input"]').clear();
     cy.get('[aria-label="Name-input"]').type("New Value Name");
     cy.get('[aria-label="Name-submit-edit"]').click();
-    cy.wait("@postEnvEdit");
-    cy.wait("@getEnv");
     cy.get('[aria-label="Name-value"]').should("contain", "New Value Name");
     //change Description value
-    cy.get('[aria-label="Description-toggle-edit"]').click();
+    cy.wait(1000);
+    cy.get('[aria-label="Description-toggle-edit"]:enabled').click();
```
Comment (LauraStordeur) [inline cypress/e2e/scenario-1-environment.cy.js:161]: cy.get('[aria-label="Description-toggle-edit"]:enabled', { timeout: 20000 }).click();

Comment (matborowczyk) [reply cypress/e2e/scenario-1-environment.cy.js:161]: I tried even with 30s timeout it didn't work

File: cypress/e2e/scenario-1-environment.cy.js (hunk 1 above)
Comment (matborowczyk) [reply cypress/e2e/scenario-1-environment.cy.js:116]: unfortunately it happened to be the other way round,I don't know why, but cy.wait resolved my issues and I didn't have any problems with flakiness so far

**Review by LauraStordeur — APPROVED** [review]

**Pull request closed without merging**